examples/simple_gpt2.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
```

Comment:
note: this breaks `JIT=1 GPU=1 PYTHONPATH=. python examples/simple_gpt2.py --prompt="Hello." --count=10 --temperature=0 --timing`